.devcontainer/devcontainer.json

```diff
@@ -1,41 +1,28 @@
 {
     "name": "mrc-dev",
     "build": {
-        "context": "..",
-        "dockerfile": "${localWorkspaceFolder}/Dockerfile",
-        "target": "development",
-        "args": {
-            "USERNAME": "${localEnv:USER}",
-            "USER_ID": "${localEnv:UID}"
-        }
+        "dockerfile": "Dockerfile"
     },
-    // todo(252)
-    // new capability with vs code 1.74 (November 2022 Release)
-    // https://github.com/microsoft/vscode-docs/blob/main/remote-release-notes/v1_74.md#gpu-host-requirement
-    // enable this after more testing is conducted
-    // "hostRequirements": {
-    //     "gpu": true
-    // },
+
     "runArgs": [
         "--gpus=all",
-        "--runtime=nvidia",
-        "--network=host",
-        "--cap-add=SYS_PTRACE",
-        "--cap-add=SYS_NICE",
-        "--security-opt",
-        "seccomp=unconfined"
```

Comment:
sys_nice is needed for huge pages
sys_ptrace and seccomp=unconfied is needed for debug
network=host is needed for some ucx transports